### ISSUE TITLE
Button should support href and target props

### ISSUE THREAD

**Issue by thabti** [quran/common-components#23]
Bootstrap button has a prop for `href` which would style an `<a>` like a button and apply the `href` value.

**Comment by mmahalwy** [comment]
hmm got to careful with this and react-router

**Comment by thabti** [comment]
Yeah - anything I should be mindful of? There is always the onClick escape hatch. And we might eventually provide a way to support styling <Link>.

**Comment by thabti** [comment]
The name will still be "Button" and not Link. I will update the storybook to Button as to not be confusing.

**Comment by mmahalwy** [comment]
`LinkContainer` would work. See: https://github.com/react-bootstrap/react-router-bootstrap/blob/master/src/LinkContainer.js

**Comment by thabti** [comment]
Yep it would I used it on the audio repo

**Comment by mmahalwy** [comment]
+1
